opentracing-api/src/main/java/io/opentracing/Span.java

```diff
@@ -111,4 +122,13 @@
      * @return this Span instance, for chaining
      */
     Span setOperationName(String operationName);
+
+    /**
+     * DEPRECATED
+     **/
+    Span log(String eventName, /* @Nullable */ Object payload);
```

Comment:
if deprecated, probably better to say what the expected replacement is. We can't ignore that most dapper systems have a plain single-arg event/annotation type. This leaves someone with a mysterious question we we now suggest logging with maps isn't only an option, but the preferred (knowing it is very rarely used in the wild)